src/components/tooltip/types.ts

```diff
@@ -0,0 +1,58 @@
+import { Position } from '@reach/tooltip'
+import { ReactElement, ReactNode } from 'react'
+export interface TooltipProps {
+  /**
+   * Element that, when hovered, will display the tooltip.
+   */
+  children: ReactElement
+
+  /**
+   * Plain text, or `<ReactElement />`, to render in the tooltip.
+   */
+  label: ReactNode
+
+  /**
+   * Optional alternate content for screen readers.
+   * Useful if the label prop is not suitable for screen reader announcement,
+   * for example for icon-only tooltips.
+   */
+  ariaLabel?: string
+
+  /**
+   * Theme for light or dark mode
+   */
+  theme?: 'light' | 'dark'
+}
+
+/**
+ * A partial DOMRect, describe further within @reach/tooltip
+ */
+export type PRect = Parameters<Position>[0]
+
+export interface TooltipArrowProps {
```

Comment:
The props interface of `TooltipArrowProps` is very much an internal detail to `components/tooltip` - but felt natural to put it in the same `types.ts` file.